out.go

```diff
@@ -37,14 +38,14 @@ func Put(request PutRequest, manager Github, inputDir string) (*PutResponse, err
 
 	// Set status if specified
 	if status := request.Params.Status; status != "" {
-		if err := manager.UpdateCommitStatus(version.Commit, request.Params.Context, status); err != nil {
+		if err := manager.UpdateCommitStatus(version.Commit, request.Params.Context, status, os.ExpandEnv(request.Params.TargetURL), request.Params.Description); err != nil {
```

Comment:
hm, I'm wondering why `os.ExpandEnv()` is used on the target URL here (given that we default to using the value from`$ATC_EXTERNAL_URL`)?

Reply:
Because, for example, we have the case where we need this TargetURL:

'https://ci.test.com/projects/my-project/terraforming/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME'

and we would like to expand the env in the url. Also, we are not using the url that provides `$ATC_EXTERNAL_URL`. Following this way, we can have the two options :). What do you thing?

Reply:
Got it, thanks for explaining :)

Reply:
you're welcome! :)